docs/widgets/video-widget.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 📹 Video
+tags:
+  - Widgets
+  - Integration
+  - Configuration
+---
+
+
+The video stream Widget is a web video player for an HTML5. It supports HTML5 video and Media Source Extensions.
+It can display content from web camera, or just watching your favorite stream.
```

Comment:
```suggestion
The Video Widget allows you to play any video supported by [video.js](https://videojs.com/) on your dashboard. This can be anything from a video to a live camera feed.
```